### PR TITLE
Configs for connecting to LaunchDarkly

### PR DESCRIPTION
**Pull request by thetif**
### Description
Added code to connect to LaunchDarkly on the back and frontend. Also added some code to dispatch flag changes throughout the system so that they can be used in redux.

### Significant changes or possible side effects
I wrapped the LaunchDarkly code around Root. That shouldn't cause any issues, but just make sure you aren't seeing weird stuff in the console.

I also have some code as proof of concepts in here, so I will be removing that before or after merging, I guess.

### Feature Flags
validation: true

### Steps to manually verify this change

1. Login to eAPD
2. Create a new eAPD
3. Open the eAPD
4. On the APD Overview you should see the message VALIDATION ON at the top of the page, the fields should show errors, and in the network console, you should see that { validation: true } is being returned as part of the APDs.
5. If you have access to the flags in LaunchDarkly, you can turn the validation flag off and confirm that none of the above appears.

VALIDATION ON is verification that the react hooks are connected to LaunchDarkly on the frontend
validation errors show that the flag values are getting to redux
validation: true being returned with the apd values shows that LaunchDarkly is working in the backend

### This pull request is ready to code review when

- [x] Automated tests are updated (and all tests are passing)
- [x] Pull request has been labeled, if applicable with feature, content, bug,
      tests, refactor

### This pull request is ready to test when

- [x] Code has been reviewed by someone other than the original author

### This pull request is ready to review when the QA has

- [x] Verified the functionality related to the change
- [ ] Verified that the change works with Narrator on Windows
- [ ] Verified that the change works with VoiceOver on Mac
- [x] Verified all updated pages with the WAVE tool
- [x] Verified tab and keyboard navigation functionality

### This pull request can be merged when

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: web/src/pages/admin/fed-admin/DelegateStateAdminForm.test.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ import MockAdapter from 'axios-mock-adapter';
 import { plain as DelegateStateAdminForm } from './DelegateStateAdminForm';
 
 const defaultProps = {
-  ffy: '2022',
+  ffy: '2023',
```
Comment (tbolt) [inline web/src/pages/admin/fed-admin/DelegateStateAdminForm.test.js:13]: I think this is fine but we could also use the `thisFFY` util in `/web/src/util/index` to have this update automatically.

File: api/middleware/launchDarkly.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  try {
+    await client.waitForInitialization();
+    // Initialization complete
+  } catch (err) {
+    // Initialization failed
+  }
+};
+
+/**
+ * To use the flag:
```
Comment (tbolt) [inline api/middleware/launchDarkly.js:23]: we want to keep this in for now?

File: api/routes/apds/get.js [hunk 3]
```diff
@@ -72,6 +73,12 @@ module.exports = (
         logger.info(`id: ${req.params.id}, state: ${stateId}`);
         const apdFromDB = await getAPDByIDAndState(req.params.id, stateId);
 
+        const validation = await ldClient.variation(
```
Comment (tbolt) [inline api/routes/apds/get.js:76]: So we pass these params to LD: key name, user info stuff, and initial flag on/off (true/false)? That's nice and straightforward if I understand it correctly.

File: web/src/redux/reducers/apd.js [hunk 4]
```diff
@@ -325,7 +326,7 @@ const initialState = {
   loaded: false,
   error: '',
   selectAPDOnLoad: false,
-  adminCheck: process.env.TEALIUM_ENV !== 'prod'
+  adminCheck: false
```
Comment (tbolt) [inline web/src/redux/reducers/apd.js:329]: 🙌

**Review by tbolt — APPROVED** [review]
Good stuff

File: api/routes/apds/get.js [hunk 5]
```diff
@@ -92,8 +85,7 @@ module.exports = (
             id,
             created,
             updated,
-            state,
-            validation
```
Comment (tbolt) [inline api/routes/apds/get.js]: why take this stuff out?

**Review by tbolt — APPROVED** [review]

**Review by mirano-darren — APPROVED** [review]

File: api/routes/apds/get.js (hunk 5 above)
Comment (thetif) [reply api/routes/apds/get.js]: I only put it in for proof of concept and I already demoed it. I've created a [wiki page](https://github.com/CMSgov/eAPD/wiki/LaunchDarkly) for how to use it.

**Pull request merged**